app/jobs/solidus_bolt/application_job.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+module SolidusBolt
+  class ApplicationJob < ActiveJob::Base
```

Comment:
Why this is required?

Reply:
requested by rubocop - if I do `SolidusBolt::AddAddressJob < ActiveJob::Base` it gives me an offence saying it should inherit from `ApplicationJob`

Reply:
Can you try this ?
`SolidusBolt::AddAddressJob < ApplicationJob`

Reply:
right you are - I was basing myself of SolidusPaypal extension, but I see that removing it works too

Fixed in https://github.com/nebulab/solidus_bolt/pull/96/commits/71f1726c8360f11405ce291fa7630e61f9d5db39